common/religion/religions/deathbringer.txt

```diff
@@ -24,6 +24,20 @@
 	doctrine = doctrine_deviancy_accepted
 	doctrine = doctrine_witchcraft_accepted
 
+	#Magic
+	doctrine = doctrine_light_magic_crime
+	doctrine = doctrine_shadow_magic_accepted
+	doctrine = doctrine_disorder_magic_crime
```

Comment:
I think it would fit more if the Fel was ignored. The Scourge is originally demon controlled after all.

Reply:
Done